tests/conversion

```diff
@@ -272,6 +282,41 @@ if hasNeededAPIExtension instance_import_conversion; then
     conversion_vm vm-u24-vmdk-dir-bupvol dir "${IMAGE_PATH}" "yes"
     conversion_vm vm-u24-vmdk-zfs-bupvol zfs "${IMAGE_PATH}" "yes"
 
+    # Test virtio conversion if virt-v2v-in-place is installed and meets the minimum version
+    # requirements. Version will be satisfied from Ubuntu 24.04 runners onwards.
+    v2vMinVersion="2.3.4"
+    if ! command -v virt-v2v-in-place > /dev/null; then
```

Comment:
```suggestion
    if ! dpkg --compare-versions "${VERSION_ID}" ge 24.04 then
```

Reply:
I suppose the `command` is fine now, considering we no longer check for the virt-v2v version